### PR TITLE
Adding missing methods to VariableUtils

### PR DESCRIPTION
**Pull request by loumalouomega**
TODO: Discuss about names of the methods (specially python)

### CODE REVIEW

File: kratos/python/add_utilities_to_python.cpp [hunk 1]
```diff
@@ -106,6 +106,29 @@ void AddUtilitiesToPython()
     .def("SetVectorVar", &VariableUtils::SetVectorVar)
     .def("SetScalarVar", &VariableUtils::SetScalarVar< Variable<double> >)
     .def("SetScalarVar", &VariableUtils::SetScalarVar< VariableComponent< VectorComponentAdaptor<array_1d<double, 3> > > >)
+    .def("SetVectorVarNonHistorical", &VariableUtils::SetVectorVarNonHistorical)
+    .def("SetScalarVarNonHistorical", &VariableUtils::SetScalarVarNonHistorical< Variable<double> >)
+    .def("SetScalarVarNonHistorical", &VariableUtils::SetScalarVarNonHistorical< VariableComponent< VectorComponentAdaptor<array_1d<double, 3> > > >)
+    .def("SetVariable", &VariableUtils::SetVariable< bool >)
```
Comment (loumalouomega) [inline kratos/python/add_utilities_to_python.cpp]: Should the non historical values have a methods just called SetVariable?, and then a different name for the historical variables (just the nodes have historical variables)

**Comment by loumalouomega** [comment]
@RiccardoRossi PING

**Comment by loumalouomega** [comment]
@RiccardoRossi PING

File: kratos/utilities/variable_utils.cpp [hunk 2]
```diff
@@ -33,8 +33,28 @@ void VariableUtils::SetVectorVar(
...
+        noalias(it_node->FastGetSolutionStepValue(rVariable)) = Value;
+    }
+    
+    KRATOS_CATCH("")
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void VariableUtils::SetVectorVarNonHistorical(
```
Comment (rubenzorrilla) [inline kratos/utilities/variable_utils.cpp]: I would call it SetNonHistoricalVectorVar.

File: kratos/utilities/variable_utils.h [hunk 3]
```diff
@@ -103,24 +127,86 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
...
+    
+    /**
+     * Sets the nodal value of a scalar variable non historical
+     * @param rVariable reference to the scalar variable to be set
+     * @param Value Value to be set
+     * @param rNodes reference to the objective node set
+     */
     template< class TVarType >
-    void SetScalarVar(
+    void SetScalarVarNonHistorical(
```
Comment (rubenzorrilla) [inline kratos/utilities/variable_utils.h]: Same

**Comment by loumalouomega** [comment]
@RiccardoRossi PING

File: kratos/utilities/variable_utils.cpp (hunk 2 above)
Comment (loumalouomega) [reply kratos/utilities/variable_utils.cpp]: @rubenzorrilla Let's wait what @RiccardoRossi

**Review by RiccardoRossi — COMMENTED** [review]
I am fine with the commit. I agree with @rubenzorrilla 's comment on the name. So i let him approve this

**Comment by loumalouomega** [comment]
OK, I will change it

**Comment by loumalouomega** [comment]
Done

**Comment by loumalouomega** [comment]
Besides I just discover you cannot access in python to the components of a variable fro conditions and elements, I will add it in a different PR

**Comment by loumalouomega** [comment]
I cannot add a GetValue for components in python for conditions and elements, so avoid my previous comment

**Comment by loumalouomega** [comment]
Ping

**Comment by loumalouomega** [comment]
imagen

**Comment by loumalouomega** [comment]
PING

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]
Please check Codacy suggestion. I'm approving as soon as you fix that.

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**